src/core/exception.d

```diff
@@ -88,8 +90,24 @@ class FinalizeError : Error
 
     override char[] toString()
     {
-        auto other = super.next ? super.next.toString : "unknown";
-        return "An exception was thrown while finalizing an instance of class " ~ info.name ~ " :: "~other;
+        char[] chained_exception;
+
+        if (super.next)
+        {
+            char[16] line_buf;
+            cast(void)snprintf(line_buf.ptr, line_buf.length,
```

Comment:
No, is fine with me, I was just curious if it was required for some reason or if it was just purely personal preference. I also like better explicit than implicit.